### ISSUE TITLE
Mark Deliveries as Outbreak

### ISSUE THREAD

**Issue by bfrgottscma** [SiLeBAT/fcl-client#765]
As a user, I would like to be able to mark deliveries as outbreak, e.g. if I have a food item which was tested positively for the outbreak strain. Similar to the approach for stations, 
[ ] a right-click on a delivery (or multiple selected deliveries) opens a context menu and "Mark as outbreak" can be selected.
[ ] This delivery is shown in red
[ ] Weight is set 1 for this delivery and scores of the network are recalculated
[ ] The legend shows a new entry for Outbreak in the delivery section
[ ] A default Highlighting rule should be created for deliveries marked as "Outbreak" (is already present but should be checked for functionality)
[ ] Right-clicking on a delivery that has been marked as outbreak opens a context menu where you can "Unmark as Outbreak"
[ ] Right-click in the canvas triggers a context menu where you can "Clear Outbreak Deliveries"
[ ] Unmarking/Clearing outbreak deliveries recalculates the score and removes the respective entry from the legend while keeping the respective default highlighting rule.

[ ] It is possible to have a mixture of stations and deliveries that are set as outbreak

**Comment by afalenski** [comment]
Tested: See ticket Boxes.
- Delivery highlighting "Outbreak" works.

- **Legend:** In the highlighting conditions list Outbreak is at the first position in delivery and station highlightings. However, if only a deliveries are marked as outbreak, the entry "Outbreak" is the last entry in the legend. If you add an outbreak station, it becomes the first entry. Outbreak should always at the same position in the legend as in the highlighting conditions list, especially in this scenario when a station highlighting does not compete with the delivery highlighting because the station highlighting "Outbreak" is not active.
@bfrgottscma Do you agree?
- Note 1: How about connecting default station and delivery highlightings for the legend? Their position in the highlighting conditions list is not relevant, but if the user would like to edit the order in the legend, one might wonder why nothing happens when changing the order in the delivery conditions. With connecting I mean that setting "Outbreak" at postion 3 in the delivery highlightings, it also becomes position 3 in the statio highlighting conditions list. OR how about combining the Highlighting tabs: There would be no sub-tabs "Stations" and "Deliveries", but only one list with two additional columns containing checkboxes for station and delivery next to each entry, Thus, a colour needs to be defined only once for station and delivery highlightings. When something is not possible, e.g. Kill Contamination" for delivery, the checkbox is greyed out. The same for the more complicated highlighting conditions using one or more properties: If score, weight and/or outbreak is used for a highlighting conditions, both checkboxes can be checked, if another is used (e.g. address), it can only be used for stations.
- Note 1a: "Product" in the deliveries list should be named "Name" to allow a name-based highlighting condition for both station and delivery.
- Note 2: There is a delivery highlighting called '"Kill Contamination", but you cannot activate it in the graph because there is no entry in the context menu for this.

**Comment by bfrgottscma** [comment]
Regarding @afalenski's comment on the legend: I agree.

Note 1+1a: For now there will be no combination of highlighting tabs for staions and deliveries as both features are functional and including new features instead of revising old ones is more urgent.

Note 2: There should be a separate ticket for this issue.
